src/components/Events/VolunteerImpactSummary.jsx

```diff
@@ -0,0 +1,112 @@
+import { Box, Flex, Text } from '@chakra-ui/react';
+import { useEffect, useState, useContext } from 'react';
+import UserContext from '../../utils/UserContext';
+import Backend from '../../utils/utils';
+import DataCard from './DataCard';
+import { TotalEventsIcon, TotalTrashIcon, LargestTrashIcon } from '../../Assets/impact_summary/ImpactSummaryIcons';
+
+const VolunteerImpactSummary = () => {
+  const [events, setEvents] = useState(0);
+  const [total, setTotalPounds] = useState(0);
+  const [largestItem, setLargestItem] = useState(0);
+
+  const { user, updateUser } = useContext(UserContext);
+
+  useEffect(() => {
+    updateUser();
+    console.log(user);
+    getData();
+  }, []);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has missing dependencies: 'getData', 'updateUser', and 'user'. Either include them or remove the dependency array.